apps/src/aichat/views/modelCustomization/ExampleTopicsInputs.tsx

```diff
@@ -29,6 +32,17 @@ const ExampleTopicsInputs: React.FunctionComponent<{
     [dispatch]
   );
 
+  const validationAlert = useMemo(() => {
```

Comment:
Nit: This definitely works! But it reminds me of a comment I received from @thomasoniii at https://github.com/code-dot-org/code-dot-org/pull/60483#discussion_r1722172240.
I wonder if `ValidationAlert` could be defined as an internal component within `MultiInputCustomization` and perhaps the prop that is passed is whether we want to display the `ValidationAlert` or not if not read-only.
I had to kinda think through/remember where else `MultiInputCustomization` is used (in `RetrievalCustomization`).

Reply:
Chatted offline and going to do this refactor as a quick follow-up! It's a really good idea and will be a good little chunk of work I can do on my flight later :D